### ISSUE TITLE
Implement registration for tenants

### ISSUE THREAD

**Issue by rembrandtreyes** [mentorient/mentorient#71]
We should create a tenant login page as well for the system. The registration form can be either in the:
http://www.mentorient.com/Account/Register page

Or

We can split the register button on the home page and label them Property Owner | Tenant

Or the Register link can have a drop-down menu that will display the two.

There are plenty of ideas to go around, would like to get some feedback from other contributors before we proceed since this will involve multiple pieces to be created.

**Comment by drazenp** [comment]
Is it possible to create tenant profile without property (to find one?)?

If tenant cannot exists without property owner we could add `invite` button next to each tenant. Property owner could send invitation (we need min email of the tenant). Tenant can click on the link in the email; redirect to page to add password; redirect on the page to add additional details (name, phone,...) or some other page. In this case we don't need separate register page or login. We could decide after successful login where user will be redirected.

Just thinking, does it make sense to allow property owner to create property and then add tenant to the property. It could easily happen to have more tenants in one hours. Not sure how this is working :)

**Comment by m-henderson** [comment]
@rembrandtreyes @drazenp this is very interesting. have you guys given this any more thought? 

also, just to make sure this is clear, are you guys wanting the tenant to be able to login to their profile, see their stats, pay bill, ect?

**Comment by rembrandtreyes** [comment]
@m-henderson @drazenp Yes the tenant log in would work as a way for the tenants to see their bills and use that portal to pay for them as well.

So how the process will work, the property owner will enter their resident's information on their end and then residents can go create a resident account. This will require the resident and property owner to have their SSN on file. Typically the SSN is provided for things like background checks and credit checks so we should think about adding that required information to the forms as well. We probably should hold off on adding sensitive information until the website is secure.

This is what I have experienced through renting properties from leasing properties.

**Comment by m-henderson** [comment]
@rembrandtreyes @drazenp I like the idea. I think this would be awesome/fun to code up. Do you happen to know if our server needs to be PCI compliant for SSN? I don't think it does.

**Comment by rembrandtreyes** [comment]
@m-henderson @drazenp So I do not think we have to be PCI compliant if we are just storing their SSN. However, if we are allowing the tenants to pay via debit card or credit then we would need to be PCI compliant unless the direction is to use a service like Stripe. There's is a lot more about data security that I am not too familiar with so if anyone else has anything to add that I am missing please let me know.
